Makefile

```
@@ -118,9 +149,19 @@ TOOLS_ELC_FILES := $(subst .el,.elc,$(TOOLS_EL_FILES))
+#   The GitHub CI/CD is identified by the presence of the GITHUB_WORKSPACE
+#   environment variable is defined.
 
 .PHONY:	all
-all: main tools
+
+ifeq ($(GITHUB_WORKSPACE),)
+# On user's systems
+all: main tools tools/s7xref test
+else
+# On GitHub CI systems
+all: main tools test
```

checkmake (0.3.2)

[warning] Target "all" defined multiple times (lines 160 and 163).

(uniquetargets)